src/containers/my-resources/attachments-container/AttachmentsContainer.constants.tsx

```diff
@@ -0,0 +1,56 @@
+import Typography from '@mui/material/Typography'
+
+import IconExtentionWithTitle from '~/components/icon-extention-with-title/IconExtentionWithTitle'
+import { getFormattedDate } from '~/utils/helper-functions'
+import {
+  SortEnum,
+  TableColumn,
+  Attachment,
+  AdditionalPropsInterface
+} from '~/types'
+import { styles } from '~/containers/my-resources/attachments-container/AttachmentsContainer.styles'
+
+export const columns: TableColumn<Attachment>[] = [
+  {
+    label: 'myResourcesPage.attachments.title',
+    field: 'title',
+    calculatedCellValue: (item: Attachment) => {
+      return <IconExtentionWithTitle description={15} title={item.fileName} />
```

Comment:
it has to be item.size instead of mocked **15**

Reply:
i should remove it at all accroding to figma. fixed